From: Zhao Zhili <zhilizhao@tencent.com>
Date: Tue, 28 Feb 2023 18:23:00 +0000 (+0800)
Subject: avcodec/aacps_tablegen: fix build error after avutil bump
X-Git-Url: http://git.videolan.org/?p=ffmpeg.git;a=commitdiff_plain;h=46970dd1555b3e50eee48ec95c893ee9a52f7fab

avcodec/aacps_tablegen: fix build error after avutil bump

Fix tickets #10225

DECLARE_ALIGNED has been moved to mem_internal.h.

Signed-off-by: Zhao Zhili <zhilizhao@tencent.com>
Reviewed-by: Anton Khirnov <anton@khirnov.net>
(cherry picked from commit 814178f92647be2411516bbb82f48532373d2554)
---

diff --git a/libavcodec/aacps_tablegen.h b/libavcodec/aacps_tablegen.h
index 0ac4f68d68..5fdd7f0a9d 100644
--- a/libavcodec/aacps_tablegen.h
+++ b/libavcodec/aacps_tablegen.h
@@ -34,7 +34,7 @@
 #include "libavutil/common.h"
 #include "libavutil/libm.h"
 #include "libavutil/mathematics.h"
-#include "libavutil/mem.h"
+#include "libavutil/mem_internal.h"
 #define NR_ALLPASS_BANDS20 30
 #define NR_ALLPASS_BANDS34 50
 #define PS_AP_LINKS 3
